sa/storage-authority.go

```diff
@@ -637,6 +637,22 @@ func (ssa *SQLStorageAuthority) AlreadyDeniedCSR(names []string) (already bool,
 	return
 }
 
+// CountCertificatesRange returns the number of certificates issued in a specific
+// date range
+func (ssa *SQLStorageAuthority) CountCertificatesRange(start, end time.Time) (count int64, err error) {
```

Comment:
Long term, this doesn't need to be called on every certificate. A cache could be kept for a while, even in-process of the RA or SA. It's kind of a bummer to do a big ol scan on every single issuance.